app/settings.py

```diff
@@ -103,7 +104,7 @@
     },
 }
 
-if env.OSFDB_HOST:
+if env.OSFDB_HOST and not sys.argv[1:2] == ["test"]:
```

Comment:
yeah, this feels brittle... i tried an alternate fix in #68 that seemed to work (disabling django's automatic transaction handling), but then ran into another osfdb error from `makemigrations` (because my local osf postgres is still 9.x...) and decided to remove the default `OSFDB_HOST` until we have credentials migration logic using it